src/assets/js/_.js

```diff
@@ -1559,4 +1559,7 @@ module.exports = {
 
 		return count === 1 ? singular : plural;
 	},
+	getScreenWidth (screenWidth) {
+		return screenWidth || window.innerWidth;
```

Comment:
I don't think this should be a function at all, but the original comment was about why we need the condition in the first place, e.g., why would the variable be undefined inside the observers?

Reply:
Except `package-stats`, `nework-request-charts` and `network-request-map` which the observer might run on other variables before `screenWidth` get set it should work out fine, but to be honest that was my question as well. If it is a concern that `screenWidth` might not be initialized at some point we can set it in the new `screenWidthListener` function. I'll add it there and remove the fallback.